docs/developer/discounts/manual-discounts.mdx

```diff
@@ -5,8 +5,9 @@ sidebar_label: Manual Discounts
 
 ## Introduction
 Manual discounts can be applied by staff users to orders with `DRAFT` or `UNCONFIRMED` status.
-These discounts cannot be combined with other discounts and override any existing ones.
-The discount can be applied to the order to decrease its subtotal and shipping price, or to the order line to reduce its unit price.
+The discount can be applied to the order to decrease its subtotal and shipping price (manual order discount), 
+or to the order line to reduce its unit price (manual line discount). 
+These discounts cannot be combined with other discounts associated with the same object (order or line) and override any existing ones.
```

Comment:
It sounds like I cannot apply a manual discount on an order with a voucher. Maybe something like this will be better:
```suggestion
These discounts are not combined with other discounts associated with the same object (order or line), manual discount takes precedence and overrides any existing ones.
```